.grunt-config/modules-with-widgets-css.js

```diff
@@ -0,0 +1,169 @@
+const path = require( 'path' );
```

Comment:
Copied this file from Pro to handle SCSS in `modules/**/assets/scss/frontend`
The current grunt tasks in Core only catered for `assets/dev/scss/frontend/widgets`

Now the fact that these css files will be compiled in `elementor/assets/css/widget-nested-tabs.min.css`, the Improved Loading Experiment will make sure it is printed inside the widget div.

So the issue was just that modules SCSS files were never compiled.